libs/lib-client/search/README.md

```diff
@@ -0,0 +1,142 @@
+# Search (`@flogo-web/lib-client/search`)
+
+## Usage
+
+### SearchComponent
+
+1. Import the search module into your module
+
+```typescript
+// my.module.ts
+import { NgModule } from '@angular/core';
+import { SearchModule } from '@flogo-web/lib-client/search';
+
+@NgModule({
+  imports: [SearchModule],
+  declarations: [],
+  entryComponents: [],
+})
+export class MyModule {}
+```
+
+2. Use it in your component template like:
+
+```html
+<flogo-search
+  (search)="onSearchChange($event)"
+  [emitOnKey]="false"
+  placeholder="Search"
+></flogo-search>
+```
+
```

Comment:
The search module is requires the consuming modules / component implement `LocalSearch` class. Should we not mention about it in the steps?